packages/terra-functional-testing/lib/terra-dev-site/doc/asset-service/AssetService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+# Asset Service
+
+The asset service initializes a server to serve compiled assets to the testing environment.
+
+## Installation
+
+The asset service should be installed as a development dependency.
+
+```bash
+npm install --save-dev @cerner/terra-functional-testing
+```
+
+## Configuration
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+            // Asset service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
+
+A path to a webpack configuration file.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: The asset service will check the current working directory for a `webpack.config.js` file.
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          config: './path/webpack.config.js'
+        }]
+    ],
+    // ...
+};
+```
+
+### index
+
+The asset index.
+
+Type: `string`
+
+Required: `false`
+
+Default: `index.html`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          index: 'index.html',
+        }]
+    ],
+    // ...
+};
+```
+
+### locale
+
+An optional locale that will be passed into the webpack configuration as an environment variable.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          locale: 'en-US',
+        }]
+    ],
+    // ...
+};
+```
+
+### port
+
+The port the service will be hosted on.
+
+Type: `number`
+
+Required: `false`
+
+Default: `8080`
```

Comment:
Thought on making this default value different than 8080. 8080 is the default value used in webpack-dev-server. This would allow someone to kick off the build and run while also developing. Or if someone kicks of builds (like me...) and has another project's site running already, the wdio run ends up failing and hanging because of https://github.com/cerner/terra-toolkit-boneyard/issues/30